.changeset/flat-adults-trade.md

```diff
@@ -0,0 +1,9 @@
+---
+"@justeattakeaway/pie-notification": patch
+"pie-storybook": patch
+"pie-monorepo": patch
+---
+
+[Added] - isOpen, variant, compact, heading, headingLevel, hideIcon properties
```

Comment:
i think the changeset might be an old one 👀

Reply:
it ts from the latest PR, and I just added a new one containing the changes of this PR

Reply:
let's delete it then if it was part of the previous PR

Reply:
Looks like this file still needs to be deleted as it was included in the previous PR: https://github.com/justeattakeaway/pie/pull/1254/files#diff-dd3a122036d1313becc5980f175cc80d41b11f46c142fb22a3997be057a67455

Reply:
This file has been deleted.